samples/posix/philosophers/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _posix-philosophers-sample:
+
+POSIX Philosophers
+##################
+
+Overview
+********
+
+This sample implements Zephyr's :ref:`Dining Philosophers Sample <dining-philosophers-sample>` using the
+:ref:`POSIX API <posix_support>`. The source code for this sample can be found under
+:file:`samples/posix/philosophers`.
+
+Building and Running
+********************
+
+This project outputs to the console.  It can be built and executed
```

Comment:
Line wrapping looks very aggressive here (and inconsistent throughout the file)

Reply:
Fixed!